frontend/e2e-test/test/e2e/specs/5_employee/application-details.spec.ts

```diff
@@ -74,6 +79,16 @@ fixture('Application - employee application details')
       ),
       id: '0c8b9ad3-d283-460d-a5d4-77bdcbc69374'
     }
+    application4 = {
```

Comment:
Supernit: all of these should be named so that their purpose/content is obvious; `application4` says nothing but `applicationRestrictedDetails` would be clearer.

Reply:
You're correct, I improved this a little.